nx/test/nx/pytorch_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Nx.PytorchTest do
+  use ExUnit.Case, async: true
+
+  alias Nx.PytorchBackend
+
+  describe "tensor" do
+    test "add" do
+      a = Nx.tensor([[1, 2], [3, 4]], backend: PytorchBackend)
+      b = Nx.tensor([[5, 6], [7, 8.0]], backend: PytorchBackend)
+
+      c = Nx.add(a, b)
+
+      assert Nx.to_flat_list(c) == [6.0, 8.0, 10.0, 12.0]
+    end
+
+    test "dot" do
+      a = Nx.tensor([[1, 2], [3, 4]], backend: PytorchBackend)
+      b = Nx.tensor([[5, 6], [7, 8]], backend: PytorchBackend)
+
+      c = Nx.dot(a, b)
+
+      assert Nx.to_flat_list(c) == [19, 22, 43, 50]
```

Comment:
Ideally do this in the tests:

    assert Nx.dot(a, b) |> Nx.backend_transfer() == Nx.tensor([19, 22, 43, 50])

It is best to compare tensors with tensors because we will also check that the shape, names, and types are correct.